lhotse/cut/set.py

```diff
@@ -1963,7 +1963,8 @@ def file_storage_path(cut: Cut, storage_path: Pathlike) -> Path:
             # too many files in a single directory.
             subdir = Path(storage_path) / cut.id[:3]
             subdir.mkdir(exist_ok=True, parents=True)
-            return (subdir / cut.id).with_suffix(f".{format}")
+            # Adding default format to fix the issue that `cut.id` contains `.`.
+            return (subdir / (cut.id + ".wav")).with_suffix(f".{format}")
```

Comment:
I think it should be 
```suggestion
            return (subdir / (cut.id + "." + format))
```